newsrevenuehub/rev-engine
# News revenue hub  sc 3591 set hub strip publishable key as environment

#### What's this PR do?

Moves all hard-coded keys and values into a settings.js file, in which each variable points to the config vars set by Heroku and made available via process.env.  Also excludes node_module in jsconfig.json file.

#### How should this be manually tested?
Please review to ensure I'm not missing any hard coded values and check for any errors before I deploy.  We can't confirm working until these changes are merged into develop and heroku re-deploys.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
